solutions/rust/05-echo/definition.yml

```diff
@@ -0,0 +1,11 @@
+author_details:
+  name: Gregory Jensen
+  profile_url: https://github.com/thetasinner
+  avatar_url: https://github.com/thetasinner.png
+  headline: CTO, CodeCrafters
+reviewers_details:
+- name:
+  profile_url:
+  avatar_url:
+  headline:
+pull_request_url:
```

Comment:
You can use the [default file](https://github.com/codecrafters-io/course-sdk/blob/main/lib/solution_definitions_compiler.rb#L8-L20) here (my name as the author, reviewer is commented out). When we get an external reviewer to review this, they'll update it

Reply:
Pretty much - If you remove the files and just run the `compile_and_test.sh` command, it should copy in the default structure we want.